composer.json

```diff
@@ -35,5 +35,6 @@
         "branch-alias": {
             "dev-master": "2.0-dev"
         }
-    }
+    },
+    "minimum-stability": "dev"
```

Comment:
No, because those symfony dependencies have dependencies on others that mean that they'd end up loading 2.x when we wanted 3.x.